src/api/config/initializers/delayed_job_config.rb

```diff
@@ -0,0 +1,4 @@
+# esp large release tracking jobs can take a long time
+# twice the time of the default (8 hours now)
+Delayed::Worker.max_run_time = 28800
```

Comment:
This will affect all jobs, which we should avoid. If something goes wrong in a job and somehow the worker is stuck, it will taking even longer before it's finally stopped. Instead, we need to change the timeout of the jobs which are needing more time.

If we use the approach from this file, we should lower the timeout of all jobs which don't require that much time. See the [delayed_job README](https://github.com/collectiveidea/delayed_job#custom-jobs) on how to do this with `max_run_time`. Another less invasive approach would be to apply what is proposed [here](https://stackoverflow.com/questions/26179508/timeout-in-a-delayed-job). This would only be in the jobs which really need more time, the remaining jobs are unchanged.